ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/viewDecisionPage.ts

```diff
@@ -1,18 +1,183 @@
-export class ViewDecisionPage
-{
-    public hasCrmEnquiry(crmNumber: string): this {
-        cy.task("log", `Has CRM enquiry ${crmNumber}`);
+export class ViewDecisionPage {
```

Comment:
Probably needs a different name as it is viewing and editing now

Maybe just decision page?

Reply:
Test refactored to reflect the above changes.